src/modal.js

```diff
@@ -68,7 +77,9 @@ export default class extends Controller {
   }
 
   close(e) {
-    if (e) e.preventDefault();
+    if (this.preventDefaultActionClosing) {
+      if (e) e.preventDefault();
```

Comment:
Can we merge these two `if`s into one? `if (e && this.preventDefaultActionClosing) {`

Reply:
@excid3 Done. ✔️